server/migrations/20220427192638-changeConstraintsAtVersion.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+module.exports = {
+    up: queryInterface => {
+        return queryInterface.sequelize.transaction(async transaction => {
+            await queryInterface.removeConstraint(
+                'AtVersion',
+                'AtVersion_pkey',
+                { transaction }
+            );
+        });
+    },
+
+    down: queryInterface => {
+        return queryInterface.sequelize.transaction(async transaction => {
+            await queryInterface.addConstraint(
```

Comment:
Seems this should be changed from the [v6](https://sequelize.org/api/v6/class/src/dialects/abstract/query-interface.js~queryinterface#instance-method-addConstraint) version to the [v5](https://sequelize.org/v5/class/lib/query-interface.js~queryinterface#instance-method-addConstraint) as well

Reply:
Good catch!